pyproject.toml

```
@@ -3,7 +3,7 @@ line-length = 100
 
 [tool.poetry]
 name = "django-log-formatter-asim"
-version = "0.0.6"
+version = "0.0.7"
```

This is being used in the wild, in production. Let's bump the version to 1.0.0....
```suggestion
version = "1.0.0"
```